packages/core/package.json

```diff
@@ -1,62 +1,19 @@
 {
   "name": "@interactors/core",
-  "version": "1.0.0",
+  "version": "0.0.0",
   "description": "Composable page objects for components",
-  "main": "dist/cjs/index.js",
-  "browser": "dist/esm/index.js",
-  "types": "dist/index.d.ts",
-  "repository": "https://github.com/thefrontside/interactors.git",
+  "license": "MIT",
   "homepage": "https://frontside.com/interactors",
   "author": "Frontside Engineering <engineering@frontside.com>",
-  "license": "MIT",
-  "files": [
-    "dist/**/*",
-    "src/**/*",
-    "README.md"
-  ],
-  "exports": {
-    ".": {
-      "require": "./dist/cjs/index.js",
-      "default": "./dist/esm/index.js"
-    }
-  },
-  "scripts": {
-    "clean": "rm -rf dist *.tsbuildinfo",
-    "lint": "eslint \"{src,test}/**/*.ts\"",
-    "check:types": "tsc --noEmit",
-    "test:unit": "tsx --test --test-reporter spec --no-deprecation test/**/*.test.ts",
-    "test:types": "tsd -f ./types/*.test-d.ts",
-    "test": "yarn test:unit && yarn test:types",
-    "docs": "rm -rf docs && yarn typedoc --options typedoc.json",
-    "prepack": "tsc --build ./tsconfig.build.json && yarn prepack:es2015 && yarn prepack:commonjs",
-    "prepack:es2015": "tsc --project ./tsconfig.build.json --outdir dist/esm --module es2015",
-    "prepack:commonjs": "tsc --project ./tsconfig.build.json --outdir dist/cjs --module commonjs"
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/thefrontside/interactors.git"
```

Comment:
If we are keeping the package.json, do we want to add the directory? I don't know if DNT will handle that or not.
https://docs.npmjs.com/cli/v10/configuring-npm/package-json#repository

Reply:
Great idea... added.